maps/tgstation.dmm

```diff
@@ -6216,7 +6217,7 @@
 "cpC" = (/obj/structure/table,/obj/item/device/t_scanner,/obj/item/device/multitool{pixel_x = 5},/obj/item/device/radio/headset/headset_eng,/obj/item/weapon/cartridge/atmos,/obj/item/weapon/cartridge/atmos,/obj/machinery/atmospherics/pipe/manifold/scrubbers/visible{dir = 4},/obj/item/device/rcd/rpd,/obj/item/device/rcd/rpd,/turf/simulated/floor,/area/engineering/atmos)
 "cpD" = (/obj/machinery/atmospherics/pipe/simple/yellow/visible,/turf/simulated/floor,/area/engineering/atmos)
 "cpE" = (/obj/machinery/atmospherics/pipe/simple/cyan/visible,/turf/simulated/floor,/area/engineering/atmos)
-"cpF" = (/obj/machinery/computer/general_air_control/large_tank_control{frequency = 1441; input_tag = "n2o_in"; name = "Nitrous Oxide Supply Control"; output_tag = "n2o_out"; sensors = list("n2o_sensor" = "Tank")},/obj/machinery/atmospherics/pipe/simple/filtering/visible,/turf/simulated/floor{icon_state = "escape"; dir = 4},/area/engineering/atmos)
+"cpF" = (/obj/machinery/atmospherics/trinary/filter{dir = 1; filter_type = 4; icon_state = "intact_on"; name = "Gas filter (N2O tank)"; on = 1; target_pressure = 4500},/obj/machinery/camera{c_tag = "Atmospherics East"; dir = 8},/turf/simulated/floor{icon_state = "escape"; dir = 6},/area/engineering/atmos)

```

Comment:
This is not a new instance. All filters are already instances because they are named. Same with the computers